src/lib.rs

```diff
@@ -384,6 +384,11 @@ mod tests {
         log("package_test");
 
         let test_package = project_root().join("crates").join("neon-runtime");
-        run("cargo package", &test_package);
+
+        if std::env::var("CI") == Ok("true".to_string()) {
```

Comment:
It was a little annoying having an unrelated test fail during development. Disabling the uncommitted changes check outside of CI seemed like a reasonable compromise.